skutil/h2o/java_templating/templater.py

```diff
@@ -0,0 +1,64 @@
+from jinja2 import Template, Environment, FileSystemLoader
+from sklearn.externals import six
+from abc import ABCMeta
+import os
+
+
+class Templater(six.with_metaclass(ABCMeta, object)):
```

Comment:
Since it won't always be a binomial response, we should initialize an instance of the templater that takes a response type, IMO.